README.md

```diff
@@ -412,6 +412,10 @@ Samples are in the [`samples/`](https://github.com/googleapis/java-storage/tree/
 
 To get help, follow the instructions in the [shared Troubleshooting document][troubleshooting].
 
+## Transport
+
+Cloud Storage uses gRPC for the transport layer.
+
```

Comment:
This is only partially true, and is currently in preview. The default transport is JSON over HTTP. How do we exclude this from being added to the readme right now.